configs/components/oifs/oifs.env.yaml

```diff
@@ -326,6 +326,7 @@ compiletime_environment_changes:
         # compilers and compile switches
         OIFS_FC: '$FC'
         OIFS_FFLAGS: '"-r8 -fp-model precise -align array32byte -O3 -qopenmp -g -traceback -convert big_endian -march=core-avx2 -mtune=core-avx2"' # -qoverride-limits -fast-transcendentals -m64  -fma -pc64"'
+        #OIFS_FFLAGS: '"-r8 -fp-model precise -align array32byte -O0 -check all,noarg_temp_created,bounds,uninit -ftrapuv -qopenmp -g -traceback -convert big_endian -march=core-avx2 -mtune=core-avx2"' 
```

Comment:
Is this added for a reason or can be just removed?

Reply:
These are debug flags that i used. I think its good to have them as a comment.

Reply:
Agreed. While it does "pollute" the file and one could go back into the git history to recover it; that often is harder for people than just swapping out a comment.